src/etc/inc/shaper.inc

```diff
@@ -3874,14 +3875,14 @@ EOD;
 			null,
 			$mask['bits'],
 			array_combine(range(32, 1, -1), range(32, 1, -1))
-		))->setHelp('IPv4 mask bits' . '<br />' . '255.255.255.255/?');
+		))->setHelp('IPv4 mask bits%1$s%2$s', '<br />', '255.255.255.255/?');
 
 		$group->add(new Form_Select(
 			'maskbitsv6',
 			null,
 			$mask['bitsv6'],
 			array_combine(range(128, 1, -1), range(128, 1, -1))
-		))->setHelp('IPv6 mask bits' . '<br />' . '<span style="font-family:consolas">ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff/?</span>');
+		))->setHelp('IPv6 mask bits%1$s%2$s', '<br />', '<span style="font-family:consolas">ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff/?</span>');
```

Comment:
As a separate issue, why is a different font chosen for the IPv6 address example?